src/stores/game.js

```diff
@@ -98,6 +98,8 @@ export const useGameStore = defineStore('game', {
     discarding: false,
     waitingForOpponentToDiscard: false,
     lastEventDiscardedCards: null,
+    //fives
+    discardingTwo: false,
```

Comment:
Why is this called discardingTwo if you discard one card? Is it because it's the second discard variable?

Reply:
I caught this earlier, made a mental note, and then lost that mental note. Possibly just a case of spaghetti brain lol